packages/airnode-admin/src/cli.ts

```diff
@@ -510,15 +512,16 @@ yargs
     { 'airnode-mnemonic': airnodeMnemonic },
     async (args) => {
       const airnodeAddress = await admin.deriveAirnodeAddress(args['airnode-mnemonic']);
-      console.log(`Airnode address: ${airnodeAddress}`);
+      logger.log(`Airnode address: ${airnodeAddress}`);
     }
   )
   .demandCommand(1)
   .strict()
   .fail((message, err) => {
-    if (message) console.log(message);
-    else if (err instanceof Error) console.log(`Command failed with unexpected error:\n\n${err.message}`);
-    else console.log(`Command failed with unexpected error:\n\n${err}`);
+    if (message) logger.log(message);
+    else {
```

Comment:
This is a weird way of writing this - braces for the `else` only because the line is too long. Can we make it consistent?

Reply:
I dislike one-line ifs anyway (ternaries excluded).